src/Microsoft.VisualStudio.ProjectSystem.Managed/Microsoft.VisualStudio.ProjectSystem.Managed.csproj

```diff
@@ -29,6 +29,13 @@
   <ItemGroup>
     <Folder Include="Properties\" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Tpl.Dataflow" Version="4.5.24" />
```

Comment:
Should these just be added [here](https://github.com/dotnet/project-system/blob/master/build/Targets/References.targets#L89)? Also shouldn't some of these already be pulled in MS.VS.ProjcetSystem should come in because we include the MS.VS.ProjectSystem.SDK package which should pull in Tpl.Dataflow, composition and threading. Do you just need to update the version there maybe?